x-pack/plugins/security/common/licensing/license_service.ts

```diff
@@ -92,7 +92,7 @@ export class SecurityLicenseService {
       };
     }
 
-    const isLicensePlatinumOrBetter = rawLicense.isOneOf(['enterprise', 'platinum', 'trial']);
+    const isLicensePlatinumOrBetter = rawLicense.hasAtLeast('enterprise');
```

Comment:
Since `platinum` has a [lower ordinal value](https://github.com/elastic/kibana/blob/520dd7c93ec888b267eae9b4a9f31747f940175c/x-pack/plugins/licensing/common/types.ts#L18) than `enterprise`, should this instead check against `platinum` instead of `enterprise`?

```suggestion
    const isLicensePlatinumOrBetter = rawLicense.hasAtLeast('platinum');
```

Reply:
the irony that I added this enterprise license yesterday 🙈

Reply:
@legrego updated